### PR TITLE
Add more unittest to synchronizer

### LINKED ISSUE #3018: adapt test for synchronizer: unittest and e2e-group 6 to etrog
# Rationale

The tests need to be adapted to the etrog changes

### PR DESCRIPTION
**Pull request by joanestebanr**
Closes #3018

### Reviewers

Main reviewers:

- @ARR552 
- @tclemos

### CODE REVIEW

File: state/test/forkid_dragonfruit/genesis_test.go [hunk 1]
```diff
@@ -68,7 +70,17 @@ func TestGenesisVectors(t *testing.T) {
 	}
 	for _, f := range files {
 		var tv []genesisTestVectorReader
-		data, err := os.ReadFile(f)
+		var data []byte
+		var err error
+
+		data, err = os.ReadFile(f)
```
Comment (ARR552) [inline state/test/forkid_dragonfruit/genesis_test.go]: What is the purpose of this change? I don't see any reason

File: synchronizer/common/errors.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package common
+
+import "errors"
+
+var (
+	// ErrInvalidParams is used when the object is not found
+	ErrInvalidParams = errors.New("invalid params")
+)
```
Comment (ARR552) [inline synchronizer/common/errors.go]: It is really needed to create a file to just one error? I recommend to leave it in the previous file

File: synchronizer/l1_parallel_sync/l1_data_message.go [hunk 3]
```diff
@@ -32,7 +32,8 @@ type L1SyncMessage struct {
 }
 
 type l1ConsumerControl struct {
-	event eventEnum
+	event     eventEnum
+	parameter uint64
```
Comment (ARR552) [inline synchronizer/l1_parallel_sync/l1_data_message.go:36]: what is this?? can you use a more descriptive name? maybe message?

File: synchronizer/common/errors.go (hunk 2 above)
Comment (joanestebanr) [reply synchronizer/common/errors.go]: I remove the file and returned the error to actions.go

File: state/test/forkid_dragonfruit/genesis_test.go (hunk 1 above)
Comment (joanestebanr) [reply state/test/forkid_dragonfruit/genesis_test.go]: I have revert this change to Thiago PR

File: synchronizer/l1_parallel_sync/l1_data_message.go (hunk 3 above)
Comment (joanestebanr) [reply synchronizer/l1_parallel_sync/l1_data_message.go:36]: If you refer to `parameter` is just that an parameter of the event, it depends on `event`

**Review by ARR552 — APPROVED** [review]
